pkg/api/metadata.go

```diff
@@ -22,11 +22,27 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"runtime/debug"
 
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
 	"github.com/getsentry/sentry-go"
 	"github.com/golang/glog"
 )
 
+func GetCommitSha() (string, error) {
+	var commitSha string
```

Comment:
```suggestion
```

Reply:
Thanks @machi1990 ! Just committed the suggested changes.